shopyo/modules/category/test_category.py

```diff
@@ -16,27 +19,27 @@ def test_category_dashboard_page(test_client):
     THEN check that the response is valid
     """
     # Logout and try to access the category dashboard. It should redirect
-    response = test_client.get(url_for('login.logout'), follow_redirects=True)
+    response = test_client.get(url_for("login.logout"), follow_redirects=True)
     assert response.status_code == 200
     assert b"Successfully logged out" in response.data
 
     # check request to category correctly redirects to login page
     response = test_client.get(
-        url_for('category.dashboard'), follow_redirects=True)
-    assert response.status_code == 200
-    assert request.path == url_for('login.login')
+        url_for("category.dashboard"), follow_redirects=True
+    )
+    assert request.path == url_for("login.login")
```

Comment:
The remove changes are the correct one. It missing one assert line 26  `assert response.status_code == 200`